rand_distr/src/gamma.rs

```diff
@@ -124,16 +125,18 @@ where
     /// distribution.
     #[inline]
     pub fn new(shape: N, scale: N) -> Result<Gamma<N>, Error> {
-        if !(shape > N::from(0.0)) {
+        let zero = N::from(0.0).unwrap();
+        let one = N::from(1.0).unwrap();
```

Comment:
Should be able to replace this with `N::zero()` and `N::one()` (via `Zero` and `One` traits).

Reply:
It will require additional trait bounds, which I am not sure worth in this case.

Hopefully one day `num-traits` will [add](https://github.com/rust-num/num-traits/issues/47#issuecomment-645839643) `try_from`, so we will be able to remove all those `unwrap`s.

Reply:
No, `Float` requires `Num` which requires `One` + `Zero`.